src/main/java/org/jabref/gui/frame/JabRefFrame.java

```diff
@@ -428,16 +466,39 @@ private void updateTabBarVisible() {
      * Returns the currently viewed LibraryTab.
      */
     public LibraryTab getCurrentLibraryTab() {
-        if (tabbedPane.getSelectionModel().getSelectedItem() == null) {
-            return null;
-        }
-        return (LibraryTab) tabbedPane.getSelectionModel().getSelectedItem();
+        return (LibraryTab) Optional.ofNullable(tabbedPane.getSelectionModel().getSelectedItem())
+                                    .filter(tab -> tab instanceof LibraryTab).orElse(null);
     }
 
     public void showLibraryTab(@NonNull LibraryTab libraryTab) {
         tabbedPane.getSelectionModel().select(libraryTab);
     }
 
+    public void showWelcomeTab() {
+        for (Tab tab : tabbedPane.getTabs()) {
+            if (!(tab instanceof LibraryTab)) {
+                tabbedPane.getSelectionModel().select(tab);
+                return;
+            }
```

Comment:
Why this loop? There should be a Java comment why this is needed.

Maybe also JavaDoc to showWelcomeTab that it is only shown of no other tabs exist. -- Update: I think, you are checking for WelcomeTab - why is the `if` not checking for instanceOf WelcomeTab?!